src/VM/Handler/Opcode8138Handler.cpp

```diff
@@ -40,9 +42,40 @@ namespace Falltergeist
 
             void Opcode8138::_run()
             {
+                //TODO: search ContainerItemObject and CritterObject inventories recursively if it has container itemobjects in it
+                //for now recursive inventories are not yet implemented since ItemObject has no inventory vector
+                
                 Logger::debug("SCRIPT") << "[8138] [=] int item_caps_total(void* obj)" << std::endl;
-                _script->dataStack()->popObject();
-                _script->dataStack()->push(0);
+                auto object = _script->dataStack()->popObject();
+                if (auto container = dynamic_cast<Game::ContainerItemObject*>(object))
+                {
+                    for(auto item: *container->inventory())
+                    {   
+                        if(item->PID() == 519)
```

Comment:
You can't check by name, because name is loaded from translation files, it's different depending on game installation... 

I just checked the engine, you just iterate over inventory contents and compare PID with enum value of PID_BOTTLE_CAPS and also use recursion if item subtype is container.